libs/core/src/lib/datetime-picker/datetime-picker.component.html

```diff
@@ -73,6 +73,18 @@
                     (focusArrowLeft)="focusArrowLeft()"
                 ></fd-time>
             </div>
+            <div fd-popover-body-footer>
+                <div fd-bar [barDesign]="'footer'" [cosy]="true">
```

Comment:
It would be better to depend `cosy` on `!compact` flag from datetime picker instead of hardcoding it